crates/wdk-build/src/lib.rs

```diff
@@ -981,14 +981,8 @@ pub fn configure_wdk_binary_build() -> Result<(), ConfigError> {
     Config::from_env_auto()?.configure_binary_build()
 }
 
-// This currently only exports the driver type, but may export more metadata in
```

Comment:
keep this comment and actually change it to a doc comment (ie. triple backslash)